Resources/doc/index.rst

```diff
@@ -9,7 +9,7 @@ Admin Bundle
 * `SonataDoctrinePhpcrAdminBundle <https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle>`_: integrates PHPCR with the core admin bundle (early stage)
 * `SonataPropelAdminBundle <https://github.com/sonata-project/SonataPropelAdminBundle>`_: integrates Propel with the core admin bundle (early stage)
 
-The demo website can be found in http://demo.sonata-project.org (``admin`` as user and password).
+The demo website can be found at http://demo.sonata-project.org.
```

Comment:
yes but the credentials are written down there:
![screenshot 2017-03-07 10 09 59](https://cloud.githubusercontent.com/assets/995707/23649290/44bcc084-031e-11e7-992b-6b377ca17804.png)